wavefront/server/modules/user_management_module/user_management_module/services/email_service.py

```
@@ -92,14 +107,28 @@ def send_email(self, subject: str, body: str, email_id: str) -> bool:
+                    '@odata.type': '#microsoft.graph.fileAttachment',
+                    'name': attachment['filename'],
+                    'contentType': attachment.get(
+                        'mime_type', 'application/octet-stream'
+                    ),
+                    'contentBytes': base64.b64encode(
+                        attachment['content_bytes']
+                    ).decode('utf-8'),
+                }
+                for attachment in attachments
+            ]
+        response = requests.post(url, headers=headers, json=message_data)
```

Ruff (0.15.6)

[error] Probable use of `requests` call without timeout

(S113)